components/camel-dhis2/camel-dhis2-api/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>camel-dhis2-parent</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>camel-dhis2-api</artifactId>
+    <name>Camel DHIS2 API</name>
+    <description>Camel DHIS2 API</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <firstVersion>3.21.0</firstVersion>
```

Comment:
This should be 4.0.0-SNAPSHOT

Reply:
In that case no, it should be 3.21.0, but only after 3.21.0 will be release, so for the moment it should be 4.0.0-SNAPSHOT, until we release